src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -178,6 +178,10 @@
   <data name="HowToOpenRun.Text" xml:space="preserve">
     <value>Ctrl+Click to follow link</value>
   </data>
+  <data name="InvalidUri" xml:space="preserve">
+    <value>Invalid URI</value>
+    <comment>Whenever we encounter an invalid URI or URL we show this string as a warning.</comment>

```

Comment:
```suggestion
    <comment>Whenever we encounter an invalid URI or URL we show this string as an error.</comment>
```
nit

Reply:
Hmm I feel like "as a warning" is better in this case. 🤔